ragna/deploy/_ui/central_view.py

```diff
@@ -117,130 +86,156 @@ class RagnaChatCopyIcon(ReactiveHTML):
-        role = self.msg_data["role"] if "role" in self.msg_data else None
-        self.css_classes = ["chat-entry", f"chat-entry-{role}"]
+        try:
+            organization, model = user.split("/")
+        except ValueError:
+            organization = ""
+            model = user
 
-    @classmethod
-    def get_avatar(cls, role, user) -> str:
-        if role == "system":
+        if organization == "Ragna":
             return "imgs/ragna_logo.svg"
-        elif role == "user":
-            # FIXME: user needs to be dynamic based on the username that was logged in with
-            return "👤"
-        elif role == "assistant":
-            # FIXME: This needs to represent the assistant somehow
-            if user == "Ragna/DemoAssistant":
-                return "imgs/ragna_logo.svg"
-            elif user.startswith("OpenAI/gpt-3.5"):
-                return pn.chat.message.GPT_3_LOGO
-            elif user == "OpenAI/gpt-4":
-                return pn.chat.message.GPT_4_LOGO
-
-            return "🤖"
-
-        # should never happen
-        return "?"
-
-    @classmethod
-    def chat_entry_value_renderer(cls, txt, role):
-        markdown_css_classes = []
-        if role is not None:
-            markdown_css_classes = [
-                f"chat-entry-{role}",
-            ]
+        elif organization == "OpenAI":
+            if model.startswith("gpt-3"):
+                return "https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/ChatGPT_logo.svg/1024px-ChatGPT_logo.svg.png?20230318122128"
+            elif model.startswith("gpt-4"):
+                return "https://upload.wikimedia.org/wikipedia/commons/a/a4/GPT-4.png"
+        elif organization == "Anthropic":
+            return "https://upload.wikimedia.org/wikipedia/commons/1/14/Anthropic.png"
+
+        return model[0].upper()
 
+    def _render(self, content: str) -> pn.pane.Markdown:
         return pn.pane.Markdown(
-            txt,
-            css_classes=markdown_css_classes,
-            stylesheets=[markdown_table_stylesheet],
+            content,
+            css_classes=["message-content", f"message-content-{self.role}"],
+            stylesheets=ui.stylesheets(
+                (
+                    "table",
+                    {
+                        "margin-top": "10px",
+                        "margin-bottom": "10px",
+                    },
+                ),
+                (
+                    (
+                        ":host .message-content-system",
+                        ":host .message-content-assistant",
+                    ),
+                    {
+                        "background": "none",
+                        "border": "lightgray",
+                        "border-style": "solid",
+                        "border-width": "thin",
+                    },
+                ),
+                (
+                    ":host .message-content-assistant",
+                    {"background": "lightgray"},
+                ),
```

Comment:
This is the only part remaining that I'm not able to solve. For some reason this is not properly applied. @pierrotsmnrd could you have a look?